src/zenml/integrations/gcp/__init__.py

```diff
@@ -72,13 +77,17 @@ def flavors(cls) -> List[Type[Flavor]]:
             GCPImageBuilderFlavor,
             VertexOrchestratorFlavor,
             VertexStepOperatorFlavor,
+            VertexModelDeployerFlavor,
+            VertexAIModelRegistryFlavor,
         )
 
         return [
             GCPArtifactStoreFlavor,
             GCPImageBuilderFlavor,
             VertexOrchestratorFlavor,
             VertexStepOperatorFlavor,
+            VertexAIModelRegistryFlavor,
```

Comment:
Can this also be called `VertexModelRegistry` without the AI, to keep it consistent with all the other flavors?